src/containers/app/OrganizationButton.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * @flow
```

Comment:
You're not actually using flow type annotations, so you don't need to add the `@flow` at the top.